README.md

```diff
@@ -112,9 +112,9 @@ or, if your webpack config file is not in the default location:
 
 This plugin contains all of the rules available in:
 
-* [ESLint](http://eslint.org/): 3.17.1
-* [eslint-plugin-react](https://github.com/yannickcr/eslint-plugin-react): 6.10.0
-* [eslint-plugin-react-native](https://github.com/intellicode/eslint-plugin-react-native): 2.3.1
+* [ESLint](http://eslint.org/): 3.17.0
```

Comment:
Umm, we didn't.  Should be 3.19.0.  Thanks for catching that; I'll update the PR when I get a chance.